medusa/search/backlog.py

```diff
@@ -49,6 +49,7 @@ def __init__(self):
         self.amActive = False
         self.amPaused = False
         self.amWaiting = False
+        self.force = False
```

Comment:
@labrys Yes. Forced because user clicked the force button on manage searches. I also added a note there